distributed/protocol/serialize.py

```diff
@@ -179,7 +180,7 @@ def msgpack_loads(header, frames):
 
 
 def serialization_error_loads(header, frames):
-    msg = "\n".join([ensure_bytes(frame).decode("utf8") for frame in frames])
+    msg = "\n".join([codecs.decode(frame, "utf8") for frame in frames])
```

Comment:
Maybe also `errors="ignore"` here, just in case.

Reply:
As an interesting side note, we may have a bug in `serialize` ( https://github.com/dask/distributed/issues/6284 )